src/zenml/integrations/evidently/steps/evidently_profile.py

```diff
@@ -137,13 +138,48 @@ def entrypoint(  # type: ignore[override]
             comparison_dataset: a Pandas DataFrame of new data you wish to
                 compare against the reference data
             config: the configuration for the step
+            ignored_columns: columns to ignore while drift detection
+
+        Raises:
+            ValueError: If ignore cols is of incorrect type or emptylist
+                or has duplicate features or if one or more features is not found
+                in either reference/comparison datasets
 
         Returns:
             profile: dictionary report extracted from an Evidently Profile
               generated for the data drift
             dashboard: HTML report extracted from an Evidently Dashboard
               generated for the data drift
         """
+        if ignored_columns is not None:
+            if not (isinstance(ignored_columns, list)):
+                raise ValueError(
+                    f"Expects a list of features but got type {type(ignored_columns)}"
+                )
+
+            if len(ignored_columns) == 0:
+                raise ValueError(
+                    "Expects a list of features or None but got empty list"
+                )
+
+            if sorted(set(ignored_columns)) != sorted(ignored_columns):
+                raise ValueError("Duplicate features found")
```

Comment:
I think maybe these two could be ignored. 

- For when the list of features is empty, then basically we will do nothing, so no need to specify a minimum number of items, I think.
- I think we can handle any duplication on our end, perhaps, so that we don't raise unnecessarily large numbers of errors here. (We could remove duplicates from the array, or convert it to a set and then back to a list etc etc.